### ISSUE TITLE
Mustang has no repair/sell functions

### ISSUE THREAD

**Issue by Aeliren** [slowtorta/turtlewow-bug-tracker#4566]
Mustang, an Alliance-aligned NPC in an unlabelled cave located above Mossgrove Farm in Gilneas, has dialogue that implies that he can serve as a repair vendor. However, he has no such options.

![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/96898927/e0127c21-35e4-4677-ad67-16999bc65e92)

**Steps to reproduce the behavior with as much detail as possible:**

1. Go to Gilneas as an Alliance character.
2. Speak with Mustang in the cave above Mossgrove Farm.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**
As the database is currently offline until October 24th, I am unable to share the NPC ID page.

**Expected behavior. Describe how it should work:**
Mustang should have an option to trade with him and repair our gear.

**Comment by twowTurtReynolds** [comment]
Agreed, this either needs a vendor + repair addition, or a dialogue rewrite.

![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/151180101/5bfd460c-9a63-4f2e-97a4-61884968beb8)
